### ISSUE TITLE
Update to a newer version of PHP_CodeSniffer

### ISSUE THREAD

**Issue by jenlampton** [backdrop-contrib/coder_review#12]
Related drupal issues: 
* D7 PHPCS 3.2.x https://www.drupal.org/project/coder/issues/2953138
* D7 PHPCS 2.2.x https://www.drupal.org/project/coder/issues/2342611
* D8 PHPCS 3.3.x https://www.drupal.org/project/coder/issues/2863898

**Comment by klonos** [comment]
@hosef mentioned in https://forum.backdropcms.org/forum/phpcs-ruleset:

> There are still a few things that I know are not working yet, such as parsing of doc blocks ...

I have recently filed https://github.com/backdrop/backdrop/pull/3299/files which uses `preg_replace()` to strip out asterisks and forward slashes from docblocks. Not sure if that would help with your docblock parsing issues @hosef

**Comment by quicksketch** [comment]
I filed another PR at https://github.com/backdrop-contrib/coder_review/pull/18 that is similar to @hosef's approach. I think @hosef's PR at https://github.com/backdrop-contrib/coder_review/pull/14 is attempting to make the existing PHPCS tests compatible with PHPCS 3+, while my PR pulls in the latest coding standards from the Drupal project and replaces a lot of the current files.

Both PRs still need work because the UI for Coder Review executing PHPCS tests doesn't actually work yet. Though I noticed that the Drupal module decided to remove all of the Coder Review UI entirely. It's now just a skeleton module that holds the PHPCS rules.

For the next step, I think we should get the existing Backdrop Coder Review UI working with PHPCS 3+. Once that is in place, we can merge the new PHPCS rules, make a new branch of Coder Review Module, and then pull the coding standards into core PR in https://github.com/backdrop/backdrop-issues/issues/3213.
